src/components/ProfilePage/elements/VisibilitySelect.messages.jsx

```diff
@@ -0,0 +1,46 @@
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
+  'profile.formcontrols.who.can.see': {
+    id: 'profile.formcontrols.who.can.see',
```

Comment:
This duplicate id will (eventually) be flagged by automated tools.

Reply:
These messages have been removed from the other file. But this should reflect visibility select. Will update

Reply:
At least in this case it's just a search and replace, but I am definitely seeing the drawback of that naming convention.  :-P